libs/database/src/lib/services/user.service.v2.spec.ts

```diff
@@ -352,12 +352,11 @@ describe('UserServiceV2', () => {
     });
 
     it('should return error when castcleId exist', async () => {
```

Comment:
```suggestion
    it('should return error? when castcleId exist', async () => {
```
ใน test ไม่เห็นมี throw error เลยครับ เปลี่ยนให้สอดคล้องด้วยครับ